jgrapht-core/src/test/java/org/jgrapht/TestUtil.java

```diff
@@ -19,61 +19,104 @@
 
 import org.jgrapht.graph.*;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 /**
  * Test related utility methods.
  */
 public class TestUtil {
 
-    public static void constructGraph(Graph<Integer, DefaultEdge> graph, int[][] edges) {
-        boolean weighted = edges.length > 0 && edges[0].length > 2;
-        for (int[] edge : edges) {
-            DefaultEdge graphEdge = Graphs.addEdgeWithVertices(graph, edge[0], edge[1]);
-            if (weighted) {
-                graph.setEdgeWeight(graphEdge, edge[2]);
-            }
-        }
+  public static void constructGraph(Graph<Integer, DefaultEdge> graph, int[][] edges) {
```

Comment:
These are spurious diffs due to an incorrect indentation setting in your editor.

Reply:
I apologise for that, I accidentally must have used auto-format. My settings are not incorrect, just different from the project's defaults. I use an indentation of two spaces throughout my two new classes and two new tests. Feel free to change that after accepting the PR. Actually in existing files I tried to copy the existing way to use braces and indentation, here it really as unintended. I can fix that if you insist. In my own classes I will not. If the indentation and other code style issues keep this PR from being accepted, fine. Throw everything away, up to you. Sorry to waste your review and debate time and a lot of my own time too. Play code style police with someone else and scare them away from contributing if you think it is such a smart idea, if you please.

Reply:
We apply autoformatting en masse during each release to achieve a uniform code style, so in general we don't get hung up on it, but when your own editor autoformatting causes a diff in code that you didn't even change, that's incorrect, and definitely a waste of my time figuring out which code actually changed, and which didn't.

Reply:
I already apologised and fixed it yesterday! I said it was my unintentional mistake. To be precise, I had the diff viewer in my IDE configured to ignore whitespace because I had a situation in another project in which I needed that then I forgot to switch it back and even though I have a habit of checking all diffs before a commit, I missed it.